packages/terra-clinical-action-header/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Added
+* prop to keep the close button in small viewports
```

Comment:
Please specify the prop name that was added

Reply:
done 6eae453